app/src/main/java/net/skyscanner/backpack/demo/compose/MapStory.kt

```diff
@@ -0,0 +1,172 @@
+            onClick = { focusedMarker = 0; viewedMarkers += 0 },
+        )
+
+        BpkPriceMapMarker(
+            title = stringResource(R.string.map_marker_price_2),
+            status = markerStatus(1),
+            state = rememberMarkerState(position = MARKER_POSITION_2),
+            onClick = { focusedMarker = 1; viewedMarkers += 1 },
+        )
+
+        BpkPriceMapMarker(
+            title = stringResource(R.string.map_marker_price_3),
+            status = markerStatus(2),
+            state = rememberMarkerState(position = MARKER_POSITION_3),
+            onClick = { focusedMarker = 2; viewedMarkers += 2 },
+        )
+
+        BpkPriceMapMarker(
+            title = stringResource(R.string.map_marker_price_4),
+            status = BpkPriceMarkerStatus.Disabled,
+            state = rememberMarkerState(position = MARKER_POSITION_4),
+        )
+    }
+}
+
+@Composable
+@MapMarkersComponent
+@ComposeStory(kind = StoryKind.DemoOnly, name = "Icon")
+fun IconMapMarkerStory(modifier: Modifier = Modifier) {
+    val cameraPositionState = rememberCameraPositionState {
+        position = MAP_POSITION
+    }
+
+    var focusedMarker by remember { mutableStateOf(0) }
+    GoogleMap(cameraPositionState = cameraPositionState) {
+        BpkIconMapMarker(
+            contentDescription = stringResource(R.string.map_marker_icon_landmark),
+            icon = BpkIcon.Landmark,
+            status = if (focusedMarker == 0) BpkIconMarkerStatus.Focused else BpkIconMarkerStatus.Default,
+            state = rememberMarkerState(position = MARKER_POSITION_1),
+            onClick = { focusedMarker = 0 },
+        )
+
+        BpkIconMapMarker(
+            contentDescription = stringResource(R.string.map_marker_icon_landmark),
+            icon = BpkIcon.Landmark,
+            status = if (focusedMarker == 1) BpkIconMarkerStatus.Focused else BpkIconMarkerStatus.Default,
+            state = rememberMarkerState(position = MARKER_POSITION_2),
+            onClick = { focusedMarker = 1 },
+        )
+
+        BpkIconMapMarker(
+            contentDescription = stringResource(R.string.map_marker_icon_cafe),
+            icon = BpkIcon.Cafe,
+            status = if (focusedMarker == 2) BpkIconMarkerStatus.Focused else BpkIconMarkerStatus.Default,
+            state = rememberMarkerState(position = MARKER_POSITION_3),
+            onClick = { focusedMarker = 2 },
+        )
+
+        BpkIconMapMarker(
+            contentDescription = stringResource(R.string.map_marker_icon_landmark),
+            icon = BpkIcon.Landmark,
+            status = BpkIconMarkerStatus.Disabled,
+            state = rememberMarkerState(position = MARKER_POSITION_4),
+        )
+    }
+}
+
+@Composable
+@MapMarkersComponent
+@ComposeStory(kind = StoryKind.DemoOnly, name = "Pointer")
+fun PointerMapMarkerStory(modifier: Modifier = Modifier) {
+    val cameraPositionState = rememberCameraPositionState {
```

Comment:
Why do we want to remember camera position state?

Reply:
we don't - removed